app/presenters/service_standard_presenter.rb

```diff
@@ -13,7 +13,6 @@ def points
   def breadcrumbs
     [
       { title: "Service manual", url: "/service-manual" },
-      { title: title },
```

Comment:
The test associated with this logic seems to still be testing for a breadcrumb without a URL. It looks like it's still passing, so not sure what's going on there. Something to look into?

https://github.com/alphagov/service-manual-frontend/blob/master/test/presenters/service_standard_presenter_test.rb#L57

Reply:
fixed test